packages/dev/test-utils/package.json

```diff
@@ -23,11 +23,14 @@
     "@react-aria/ssr": "^3.0.0",
     "@react-spectrum/test-utils": "1.0.0-alpha.1",
     "@swc/helpers": "^0.5.0",
+    "@testing-library/dom": "^10.1.0",
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^15.0.7",
+    "@testing-library/user-event": "^14.4.3",
+    "jest": "^29.5.0",
```

Comment:
these ok to be dependencies instead of peerDependencies?

Reply:
yeah I think it should be fine to make these dependencies, what was the particular reason for the move though?

Reply:
having them as peer means every package which depends on test-utils must provide these (in this case through devDeps) this is less changes